.github/workflows/terraform-compatibility-matrix.yml

```diff
@@ -0,0 +1,50 @@
+name: "HashiCorp Terraform Compatibility Matrix"
+run-name: 'HashiCorp Terraform Compatibility Matrix ${{ inputs.atlas_cloud_env }}'
+
+on:
+  schedule:
+    - cron: "0 0 1-7 * 6" # runs first Saturday of the month at midnight UTC
```

Comment:
how about running this the first day of the month: 0 0 1 * *
and test suite the rest: 0 0 2-31 * *

Reply:
I prefer to run this always on weekends to:
- Avoid using resources during the week which could affect our day to day work (e.g. PR tests failing because out of capacity)
- Reduce noise caused by the tests failing because there are not enough resources

Reply:
ok